### PR TITLE
Add Orchard CMS support

### PR DESCRIPTION
**Pull request by urbanit**
Add Orchard CMS support

### CODE REVIEW

**Comment by AliasIO** [comment]
Thanks!

File: src/apps.json [hunk 1]
```diff
@@ -4131,6 +4131,16 @@
 			},
 			"website": "oracle.com"
 		},
+		"Orchard CMS": {
++			"cats": [
```
Comment (gadcam) [inline src/apps.json]: "+" I think there is a little problem..

Comment (AliasIO) [inline src/apps.json]: Not sure what that means, are they spaces?

Comment (q--) [inline src/apps.json]: @AliasIO https://github.com/AliasIO/Wappalyzer/blob/master/src/apps.json#L4135

**Comment by urbanit** [comment]
Is everything ok?

**Comment by AliasIO** [comment]
Oh, right. Looks like this was copy/pasted from a diff? I'll fix it.

**Pull request merged**